src/lakefs_spec/client_helpers.py

```diff
@@ -18,10 +25,12 @@ def commit(
     message: str,
     metadata: dict[str, str] | None = None,
 ) -> Commit:
-    diff = client.branches_api.diff_branch(repository=repository, branch=branch)
+    diff = client.branches_api.diff_branch(
+        repository=repository, branch=branch)
```

Comment:
I'm wondering where these reformats come from - I ran `ruff format`, which passed, and I reverted them back on my machine, and both versions are apparently fine.

Could you restore the original formatting in these files (`client_helpers.py` and `test_client_helpers.py`)?